youtube_dl/extractor/odnoklassniki.py

```diff
@@ -246,5 +251,10 @@ def _real_extract(self, url):
 
         self._sort_formats(formats)
 
+        if not formats:
```

Comment:
This should be checked before sorting since currently `_sort_formats` on empty formats.